src/platform/packages/private/kbn-esql-editor/src/esql_editor.tsx

```diff
@@ -257,8 +258,12 @@ export const ESQLEditor = memo(function ESQLEditor({
       const editorLeft = editorCoords.left;
 
       // Calculate the absolute position of the popover
-      const absoluteTop = editorTop + (editorPosition?.top ?? 0) + 20;
-      const absoluteLeft = editorLeft + (editorPosition?.left ?? 0);
+      const absoluteTop = editorTop + (editorPosition?.top ?? 0) + 25;
```

Comment:
How did you determine the 25 pixels?  Is there an EUI variable we could use instead?

Reply:
The height of each row is 18.5 and i had a margin of 20. I added plus 5 to give it some air. There is no variable that stores this in eui